mscore/preferenceslistwidget.h

```diff
@@ -30,15 +30,17 @@ namespace Ms {
 //---------------------------------------------------------
 //   PreferenceItem
 //---------------------------------------------------------
-class PreferenceItem : public QTreeWidgetItem, public QObject {
+class PreferenceItem : public QObject, public QTreeWidgetItem {
+
+      Q_OBJECT
 
       QString _name;
 
     protected:
       void save(QVariant value);
 
     public:
-      PreferenceItem();
+      PreferenceItem() {};
       PreferenceItem(QString name);
 
       virtual void save() = 0;
```

Comment:
Shouldn't this be called `apply()`?

Reply:
Indeed, I did not change it to not make the history of the file confusing. I'll change it since I also removed the default constructor.